server/lib/notifications.js

```diff
@@ -120,19 +119,38 @@ async function notifyUserId(UserId, activity, options = {}) {
     const parentCollective = await event.getParentCollective();
     const ics = await event.getICS();
     options.attachments = [{ filename: `${event.slug}.ics`, content: ics }];
+
     const transaction = await models.Transaction.findOne({
       where: { OrderId: activity.data.order.id, type: 'CREDIT' },
     });
-    pdf = await getTransactionPdf(transaction, user);
-    // attach pdf
-    if (pdf) {
+
+    const transactionPdf = await getTransactionPdf(transaction, user);
+    if (transactionPdf) {
       const createdAtString = toIsoDateStr(transaction.createdAt ? new Date(transaction.createdAt) : new Date());
       options.attachments.push({
         filename: `transaction_${event.slug}_${createdAtString}_${transaction.uuid}.pdf`,
-        content: pdf,
+        content: transactionPdf,
       });
       activity.data.transactionPdf = true;
     }
+
+    if (transaction.hasPlatformTip()) {
+      const platformTipTransaction = await transaction.getPlatformTipTransaction();
+      if (platformTipTransaction) {
+        const platformTipPdf = await getTransactionPdf(platformTipTransaction, user);
+        if (platformTipPdf) {
+          const createdAtString = toIsoDateStr(
+            platformTipTransaction.createdAt ? new Date(platformTipTransaction.createdAt) : new Date(),
```

Comment:
This ternary condition was there in the previous version but I didn't pay attention to it. I'm a bit surprised now, because `createdAt` is not nullable.